app/person/controllers.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios')
+
+const personService = require('../../common/services/person')
+
+module.exports = {
+  image: async (req, res) => {
+    try {
+      const imageUrl = await personService.getImageUrl(req.params.personId)
+      const response = await axios.get(imageUrl, {
+        responseType: 'arraybuffer',
+      })
+
+      res.writeHead(200, {
+        'Content-Type': response.headers['content-type'],
+      })
+      res.end(response.data, 'binary')
+    } catch (error) {
+      res.redirect('/images/person-placeholder.png')
```

Comment:
I wasn't 100% convinced on this, open to other suggestions, but the alternative was to load the image from file and serve it back.

Also, not sure if we should be storing this in some kind of config? Might be easier that way.